src/jv.c

```diff
@@ -164,7 +164,7 @@ jv jv_invalid_get_msg(jv inv) {
 
   jv x;
   if (JVP_HAS_FLAGS(inv, JVP_FLAGS_INVALID_MSG)) {
-    x = jv_copy(((jvp_invalid*)inv.u.ptr)->errmsg);
+    x = jv_unborrow(((jvp_invalid*)inv.u.ptr)->errmsg);
```

Comment:
This should not be needed.  `jv`s contained in others should never be borrowed, I'd think.

Reply:
Actually, no jv_invalid_with_msg() currently does not unborrow its errormessage. This should be changed and is a bug(for an explanation see why jv_set has to increment the refcount)

Reply:
But yeah with jv_invalid_with_msg() fixed all jv functions should be consistent in unborrowing when writing, thus there is no need to unborrow when reading